main.py

```diff
@@ -47,6 +42,70 @@ async def get_app(app_id:str):
         return {"message": "No App found for id"}
     except Exception as e:
         return {e}
+    
+
+@app.post("/app")
+async def create_app(app: App):
+    try:
+        document = jsonable_encoder(app)
+        document["authors"] = [ObjectId(author) for author in document["authors"]]
+        result = await db['apps'].insert_one(document)
+        if not result:
+            raise HTTPException(status_code=400, detail="App could not be created")
+        return {"App": f"{result.inserted_id} id created successfully!"}
+    
+    except Exception as e:
+        return {e}
+
+@app.post("/user/author")
+async def create_author(author: Author):
+    try:
+        document = jsonable_encoder(author)
+        document["apps"] = [ObjectId(app) for app in document["apps"]]
+        result = await db['authors'].insert_one(document)
+        if not result:
+            raise HTTPException(status_code=400, detail="Author could not be created")
+        
+        return {"Author": f"{result.inserted_id} id created successfully!"}
+    except Exception as e:
+        return {"message": str(e)}
+   
+
+@app.get("/authors")
+async def get_authors() -> Author | None:
```

Comment:
since we are returning **Authors** class the return type should be changed from **Author** class to **Authors** Class 

also do we need a separate model just to send a list of author model? 
we can also use **List[Author]** 
CC: @abhibansal60 @Anthony-clintan @SumukhaNadig